src/lib.rs

```diff
@@ -124,6 +126,9 @@ fn entrypoint(py: Python, m: &PyModule) -> PyResult<()> {
         py.get_type::<trustchain::TrustchainErrorException>(),
     )?;
 
+    // RemoteDevicesManager
+    m.add_class::<remote_devices_manager::RemoteDevicesManager>()?;
```

Comment:
should be in `remote_devices_manager::add_mod`

Reply:
Done